tests/integration/test_parallel_replicas_custom_key_failover/test.py

```diff
@@ -87,7 +73,7 @@ def test_parallel_replicas_custom_key_failover(
     log_comment = uuid.uuid4()
     assert (
         node1.query(
-            f"SELECT key, count() FROM {table}_d GROUP BY key ORDER BY key",
+            f"SELECT key, count() FROM cluster('{cluster}', default.test_table) GROUP BY key ORDER BY key",
```

Comment:
shouldn't distributed table be same?

Reply:
Well, I thought that we'll build a new `Cluster` representation in memory in case of `cluster()` function, but we're using an existing one. So, your point is valid. But let's keep it, since it's a typical way of using custom key mode.